app/lib/features/space/providers/space_navbar_provider.dart

```diff
@@ -69,15 +68,15 @@ final tabsProvider =
     }
   }
 
-  final spacesList = await ref.watch(relatedSpacesProvider(spaceId).future);
-  if (spacesList.isNotEmpty) {
+  final hasSpaces = await ref.watch(hasSubSpacesProvider(spaceId).future);
+  if (hasSpaces) {
```

Comment:
Look good for fine but need think in future for proper solution.

Reply:
I have an idea about that already, expect an update soon.